custom_components/ha_washdata/store_client.py

```
@@ -241,13 +296,33 @@ async def _run_query(self, sq: dict[str, Any], parent: str = "") -> list[dict[st
                         _LOGGER.warning("Store query HTTP %s (no body)", resp.status)
-                    return []
+                    return None
                 rows = await resp.json()
         except Exception as exc:  # noqa: BLE001
             _LOGGER.warning("Store query error: %s", exc)
-            return []
+            return None
         return [_decode_doc(r["document"]) for r in rows if isinstance(r, dict) and "document" in r]
 
+    async def _cached_catalog_query(
+        self, key: str, build_sq: "Callable[[], dict[str, Any]]"
```

Ruff (0.15.21)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)